project-koku/koku
# Cost-3044: OCI NULL Columns

## Jira Ticket

[COST-3044](https://issues.redhat.com/browse/COST-3044)

## Description

This change will fix the issue of null columns in OCI services summary tables by updating the sql insert statements for the respective tables.

## Testing

1. Checkout Branch
2. Restart Koku
3. Run these commands to add sources and load oci data: 
     - `make create-test-customer`
     - `make load-test-customer-data test_source=oci`
    
4. Wait for everything to load and; 
     - Run `PGPASSWORD=postgres psql postgres -U postgres -h localhost -p 15432`
     - You should see non NULL values in the respective columns for each of the tables
         ```
        reporting_oci_compute_summary_by_account_p: instance_type
        reporting_oci_compute_summary_p: unit
        reporting_oci_database_summary_p: usage_amount, unit
        reporting_oci_network_summary_p: usage_amount, unit
        reporting_oci_storage_summary_by_account_p: usage_amount
        reporting_oci_storage_summary_p: usage_amount

         ```

## Notes

...
